src/main/kotlin/com.mineinabyss.bonfire/systems/BonfireEffectSystem.kt

```diff
@@ -1,24 +1,27 @@
-package com.mineinabyss.bonfire.ecs.systems
+package com.mineinabyss.bonfire.systems
 
+import com.mineinabyss.blocky.blocky
 import com.mineinabyss.bonfire.bonfire
-import com.mineinabyss.bonfire.ecs.components.BonfireEffectArea
-import com.mineinabyss.bonfire.extensions.isBonfireModel
+import com.mineinabyss.bonfire.components.BonfireEffectArea
+import com.mineinabyss.bonfire.extensions.isBonfire
 import com.mineinabyss.geary.autoscan.AutoScan
 import com.mineinabyss.geary.systems.RepeatingSystem
 import com.mineinabyss.geary.systems.accessors.TargetScope
+import com.mineinabyss.idofront.textcomponents.miniMsg
 import com.mineinabyss.idofront.time.ticks
 import org.bukkit.Particle
+import org.bukkit.entity.ItemDisplay
 import org.bukkit.entity.Player
 
 @AutoScan
-class BonfireEffectSystem : RepeatingSystem(1.ticks) {
+class BonfireEffectSystem : RepeatingSystem(10.ticks) {
     private val TargetScope.player by get<Player>()
     private val TargetScope.effect by get<BonfireEffectArea>()
```

Comment:
When is BonfireEffectArea added and how do we untrack when player is out of the area? Or is this just a component thats added to all players with a permission or something?

Reply:
it is set when they set their spawn at a bonfire and removed whenever they remove their spawn